README.md

```diff
@@ -1,37 +1,8 @@
 # vet-clinic-database
 ## Getting Started
 
-This repository includes files with plain SQL that can be used to recreate a database:
-
-- Use [schema.sql](./schema.sql) to create all tables.
-- Use [data.sql](./data.sql) to populate tables with sample data.
-- Check [queries.sql](./queries.sql) for examples of queries that can be run on a newly created database. **Important note: this file might include queries that make changes in the database (e.g., remove records). Use them responsibly!**
-
 <a name="readme-top"></a>
 
-<!--
-HOW TO USE:
-This is an example of how you may give instructions on setting up your project locally.
-
-Modify this file to match your project and remove sections that don't apply.
-
-REQUIRED SECTIONS:
-- Table of Contents
-- About the Project
-  - Built With
-  - Live Demo
-- Getting Started
-- Authors
-- Future Features
-- Contributing
-- Show your support
-- Acknowledgements
-- License
-
-After you're finished please remove all the comments and instructions!
--->
-
-
 <!-- TABLE OF CONTENTS -->
 
 # 📗 Table of Contents
```

Comment:
Great Job documenting your README 👏🏾 

[OPTIONAL] There's one more description comment that is not removed. Kindly remove it to make the README more professional or otherwise modify the text to fit the content of the section.

![image](https://github.com/sulaimonaa/vet-clinic-database/assets/38088818/2e3123ad-eeaa-4896-a59f-f212d8a646fa)